frontend/src/pages/LearningCourse/ActionButton/ActionButton.tsx

```diff
@@ -0,0 +1,31 @@
+import { FC } from 'react';
+
+import { ActionButtonWrapper, StyledButton } from './styled';
+
+interface IProps {
+  isTestEnabled: boolean;
+  handleStartTestDialogOpen: () => void;
+  handleStageForward: () => void;
+}
+
+const enum ActionButtonText {
+  START_THE_TEST = 'Start the Text',
+  NEXT = 'Next',
+}
```

Comment:
Reuse it please instead
<img width="373" alt="image" src="https://user-images.githubusercontent.com/44285339/160670800-adfef2fb-5a5c-4308-8255-0cab6578cad5.png">